### ISSUE TITLE
Redix >= 0.10 breaks exq

### ISSUE THREAD

**Issue by danielkang674** [akira/exq#376]
:redix, "0.10.0"/"0.10.1" (v10 and above)
:exq, "0.13.2"

I get this error

```
** (EXIT from #PID<0.89.0>) shutdown: failed to start child: Exq.Manager.Server
    ** (EXIT) an exception was raised:
        ** (RuntimeError) 


====================================================================================================
ERROR! Could not connect to Redis!

Configuration passed in: [host: "x.x.x.x", port: xxxx, database: x, password: xxx]
Error: :error
Reason: :badarg

Make sure Redis is running, and your configuration matches Redis settings.
====================================================================================================

            lib/exq/manager/server.ex:405: Exq.Manager.Server.check_redis_connection/1
            lib/exq/manager/server.ex:174: Exq.Manager.Server.init/1
            (stdlib) gen_server.erl:374: :gen_server.init_it/2
            (stdlib) gen_server.erl:342: :gen_server.init_it/6
            (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
```

Had to downgrade my redix version to 0.9.3 for exq to run.

**Comment by kaushik-evani** [comment]
Any updates on this?

**Comment by ananthakumaran** [comment]
@kaushik-evani could you share more details? I upgraded to redix 0.10.2 and tests are passing https://travis-ci.org/akira/exq/builds/590705604 (there was some minor issue with application startup logic in test_helper, but that shouldn't affect others)

**Comment by danielkang674** [comment]
@ananthakumaran seems like there are some minor updates to both packages.
I'll try updating to 
:redix, "0.10.2"
:exq, "0.13.3"
and then I'll post here again after it's pushed to production.

**Comment by ananthakumaran** [comment]
Closing this as there is not enough info to debug further. We have been running exq with redix 0.10.4 without any issue on production for long time.

**Comment by Ravi-Reevoy** [comment]
I am getting this error while connecting elixir with redis, can anybody please help me with this
`** (Mix) Could not start application exq: Exq.start(:normal, []) returned an error: shutdown: failed to start child: Exq.Manager.Server
    ** (EXIT) an exception was raised:
        ** (RuntimeError) 


====================================================================================================
ERROR! Could not connect to Redis!

Configuration passed in: [[host: "localhost", port: 6379, database: 0, password: nil, name: Exq.Redis.Client, socket_opts: []]]
Error: :error
Reason: %ArgumentError{message: "expected :timeout to be an integer of :infinity, got: nil"}

Make sure Redis is running, and your configuration matches Redis settings.
====================================================================================================

            (exq 0.15.0) lib/exq/manager/server.ex:441: Exq.Manager.Server.check_redis_connection/1
            (exq 0.15.0) lib/exq/manager/server.ex:174: Exq.Manager.Server.init/1
            (stdlib 4.1.1) gen_server.erl:851: :gen_server.init_it/2
            (stdlib 4.1.1) gen_server.erl:814: :gen_server.init_it/6
            (stdlib 4.1.1) proc_lib.erl:240: :proc_lib.init_p_do_apply/3`

**Comment by Ravi-Reevoy** [comment]
Hi I got the same error when connecting elixir and redis can anybody help I am stuck on this for more than a week

**Comment by ananthakumaran** [comment]
We fixed a similar issue https://github.com/akira/exq/pull/475 recently. Please upgrade to the latest exq version 0.17.0. If the problem persists, create a new issue with the exq and redix versions along with error details.

**Comment by Ravi-Reevoy** [comment]
> We fixed a similar issue #475 recently. Please upgrade to the latest exq version 0.17.0. If the problem persists, create a new issue with the exq and redix versions along with error details.

Thanks it worked
